### PR TITLE
Ego dump

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
Something new that has never been done before.

## Why It's Good For The Game
Adds ego

## Changelog
:cl:
add: Added new ego
/:cl:

### CODE REVIEW

File: code/game/objects/items/ego_weapons/aleph.dm [hunk 1]
```diff
@@ -1120,3 +1120,117 @@
...
+	if(!user)
+		return
+	current_holder = user
+
+/obj/item/ego_weapon/mockery/dropped(mob/user)
+	. = ..()
+	current_holder = null
+
+/obj/item/ego_weapon/mockery/attack(mob/living/target, mob/living/carbon/human/user)
+	if(!CanUseEgo(user))
```
Comment (Gboster-0) [inline code/game/objects/items/ego_weapons/aleph.dm:1177]: /obj/item/ego_weapon already checks for CanUseEgo, consider removing this check

Comment (Coxswain-Navigator) [reply code/game/objects/items/ego_weapons/aleph.dm:1177]: Already being done.

File: code/game/objects/items/ego_weapons/aleph.dm [hunk 2]
```diff
@@ -1131,3 +1131,113 @@
 	if(B.safety)
 		user.remove_status_effect(STATUS_EFFECT_LCBURN)
 
+/obj/item/ego_weapon/mockery
+	name = "mockery"
+	desc = "...If I earned a name, will I get to receive love and hate from you? \
+	Will you remember me as that name, as someone whom you emotionally cared for?"
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/aleph.dm:1137]:
```suggestion
	Will you remember me as that name, as someone whom you cared for?"
```
"emotionally cared for" reads weirdly here imo. "cared for" gets the same message across and I think reads better.

File: code/game/objects/items/ego_weapons/aleph.dm [hunk 3]
```diff
@@ -712,7 +712,7 @@
 	if(!CanUseEgo(user))
 		return
 	. = ..()
-	if(current_season == "summer")
+	if(name == "summer solstice")
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/aleph.dm:715]: unnecessary change

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/ego_weapons/aleph.dm (hunk 2 above)
Comment (Coxswain-Navigator) [reply code/game/objects/items/ego_weapons/aleph.dm:1137]: It's a direct quote, but you have a point. You can't trust those translators.

File: code/game/objects/items/ego_weapons/aleph.dm (hunk 3 above)
Comment (Coxswain-Navigator) [reply code/game/objects/items/ego_weapons/aleph.dm:715]: Will fix

**Review by LanceSmites328 — APPROVED** [review]
